### PR TITLE
Fix for #2500

### PR DESCRIPTION
**Pull request by anantshri**
Probable fix for #2500 

However as i previously said someone has to build a package and inform me what errors they see. I don't have build envirnment in front of me right now and hence will need this to be reviewed by someone.

### CODE REVIEW

File: build/build-debian.xml [hunk 1]
```diff
@@ -33,7 +33,13 @@
 			<data src="debian/zap" type="file">
 				<mapper type="perm" strip="1" prefix="/usr/bin" filemode="755"/>
 			</data>
+			<data src="debian/owasp-zap.desktop" type="file">
```
Comment (thc202) [inline build/build-debian.xml]: This build file does not seem to be in use (and working), the deb package is currently built with build.xml file (in `package-linux` target).

File: build/debian/owasp-zap.desktop [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env xdg-open
+[Desktop Entry]
+Version=1.0
+Type=Application
+Terminal=false
+TryExec=/usr/bin/zap
+Exec=/usr/bin/zap
+Name=OWASP ZAP
+Icon=/usr/share/icon/zapicon.png
```
Comment (thc202) [inline build/debian/owasp-zap.desktop]: There's a typo (`icons` instead of `icon`).

**Comment by thc202** [comment]
I can see see the menu entry (with the above comments addressed).

Could you fix up the commits?

**Comment by anantshri** [comment]
@thc202 i am stuck with office work for next 8-10 hours once i am free will update the pull request.

**Comment by anantshri** [comment]
closing this aslots of stuff has changes in between let me reopen a new fresh pull request.

**Pull request closed without merging**